axon-framework/events/event-processors/README.md

```diff
@@ -519,9 +519,9 @@ public class EnqueuePolicyConfigurer {
 public class EnqueuePolicyConfigurer {
 
     @Bean
-    public ConfigurerModule configureEnqueuePolicy(String processingGroup) {
+    public ConfigurerModule configureEnqueuePolicy(EventProcessingConfigurer configurer) {
```

Comment:
The `EventProcessingConfigurer` parameter shouldn't be necessary here, due to the nature of the `ConfigurerModule`.